h2o-parsers/h2o-parquet-compat/h2o-parquet-v17-compat/testMultiNode.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Clean out any old sandbox, make a new one
+OUTDIR=sandbox
+rm -fr $OUTDIR; mkdir -p $OUTDIR
+
+# Check for os
+SEP=:
+case "`uname`" in
+    CYGWIN* )
+      SEP=";"
+      ;;
+esac
+
+function cleanup () {
+  kill -9 ${PID_1} ${PID_2} ${PID_3} ${PID_4} 1> /dev/null 2>&1
+  wait 1> /dev/null 2>&1
+  RC=`cat $OUTDIR/status.0`
+  if [ $RC -ne 0 ]; then
+    cat $OUTDIR/out.0
+    echo h2o-parquet-v17-compat junit tests FAILED
+  else
+    echo h2o-parquet-v17-compat junit tests PASSED
+  fi
+  exit $RC
+}
+
+trap cleanup SIGTERM SIGINT
+
+# Find java command
+if [ -z "$TEST_JAVA_HOME" ]; then
+  # Use default
+  JAVA_CMD="java"
+else
+  # Use test java home
+  JAVA_CMD="$TEST_JAVA_HOME/bin/java"
+  # Increase XMX since JAVA_HOME can point to java6
+  JAVA6_REGEXP=".*1\.6.*"
+  if [[ $TEST_JAVA_HOME =~ $JAVA6_REGEXP ]]; then
```

Comment:
What if the version is not part of the path ? Can this ever occur ?

Reply:
It should not. This file comes from a template. It should ideally be generated.

If you see any issue here please log a bug because it would affect all the modules.

Reply:
I was thinking about it and provided those are generated, I see no problem.